### PR TITLE
[chore](git) Some paths should be tracked by git

### PR DESCRIPTION
**Pull request by adonis0147**
# Proposed changes

~~Issue Number: close #xxx~~

## Problem summary

Some paths in the repository should be tracked by git. e.g. `tools/ssb-tools/bin`

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,7 @@ package-lock.json
 
 # ignore all sub `/bin` and `/build` directory
 **/build/
-*/**/bin/
```
Comment (xy720) [inline .gitignore:27]: We already ignore all file in output/.

File: .gitignore [hunk 2]
```diff
@@ -24,7 +24,7 @@ package-lock.json
...
+**/output/**/bin/
```
Comment (xy720) [inline .gitignore:27]: We already ignore all files in output/

Comment (adonis0147) [reply .gitignore]: Removed this line.

**Review by xy720 — APPROVED** [review]
LGTM

**Pull request merged**